test/Dialect/Seq/clock-type.mlir

```diff
@@ -0,0 +1,40 @@
+// RUN: circt-opt --lower-seq-to-sv %s | FileCheck %s
+
+hw.generator.schema @Some_schema, "Some_schema", ["dummy"]
+
+// CHECK-LABEL: hw.module.generated @generated, @Some_schema(%clock: i1, %other: i1) -> (out: i32)
+hw.module.generated @generated, @Some_schema(%clock: !seq.clock, %other: i1) -> (out: i32) attributes { dummy = 1 : i32 }
+
+// CHECK-LABEL: hw.module.extern @extern(%clock: i1, %other: i1) -> (out: i32)
+hw.module.extern @extern(%clock: !seq.clock, %other: i1) -> (out: i32)
+
+// CHECK-LABEL: hw.module @top(%clock: i1, %other: i1, %wire: i1)
```

Comment:
Check that the output has been lowered.

Reply:
I think that's what it does. FileCheck will blow up if it doesn't find the entire line in the output.

Reply:
Sorry, I meant output **port**. This will pass even on a line ending in ` -> (out: !seq.clock)` (which is wrong).

Reply:
Ah good catch!